index.css

```diff
@@ -1174,3 +1171,555 @@ textarea {
+    width: 23px;
+    height: 23px;
+    border-radius: 50%;
+    background: #ebebff;
+    position: absolute;
+    top: 22.5%;
+    left: 51.3%;
+  }
+
+  .rectangle53 {
+    width: 108.67px;
+    height: 19.44px;
+    background: #ebebff;
+    transform: rotate(-46.14deg);
+    position: absolute;
+  }
+
+  .hal-cir {
+    width: 80.48px;
+    height: 40.24px;
+    border-top-left-radius: 80.48px;
+    border-top-right-radius: 80.48px;
+    background: #6070ff;
+    transform: rotate(-156.88deg);
+    position: absolute;
+    top: 13%;
+    left: 5%;
+    z-index: -1;
+  }
+
+  .half-circle1 {
+    width: 36.9px;
+    height: 18.45px;
+    border-top-left-radius: 36.9px;
+    border-top-right-radius: 36.9px;
+    background: #6070ff;
+    transform: rotate(-176.08deg);
+    position: absolute;
+    top: 80%;
+    left: 15%;
+  }
+
+  .circle-b {
+    width: 47px;
+    height: 47px;
+    border-radius: 50%;
+    background: #6070ff;
+    position: absolute;
+    top: 80%;
+    left: 40%;
+  }
+
+  .rectangle41 {
+    width: 161.41px;
+    height: 28.87px;
+    background: #ebebff;
+    transform: rotate(23.34deg);
+    position: absolute;
+    top: 90%;
+    left: 35%;
+  }
+
+  .works {
+    width: 100vw;
+    left: 0;
+    top: 470px;
+    padding-top: 230px;
+  }
+
+  .card {
+    width: 80vw;
+    flex-direction: row;
+  }
+
+  .card img {
+    width: 50%;
+    height: 95%;
+    margin-left: 15px;
+  }
+
+  .card .info-block {
+    height: 90%;
+    width: 45%;
+
+    /* margin-left: 46px; */
+    margin-top: 10px;
+  }
+
+  .card .client {
+    width: 100%;
+  }
+
+  .client .project-title {
+    font-size: 40px;
+    line-height: 52px;
+    width: 400px;
+  }
+
+  .details {
+    width: 100%;
+  }
+
+  .details .name .project-name {
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .details .circle-1 {
+    margin-left: 20px;
+    margin-top: 7px;
+  }
+
+  .details .role {
+    width: 100px;
+  }
+
+  .details .role .project-role {
+    margin-top: 16px;
+    font-size: 18px;
+    line-height: 24px;
+    width: 200px;
+  }
+
+  .details .circle-2 {
+    margin-left: 17px;
+    margin-top: 7px;
+  }
+
+  .details .year {
+    margin-top: 5px;
+    margin-left: -10px;
+  }
+
+  .details .project-year .year {
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .summary {
+    margin-top: 50px;
+    width: 100%;
+    height: 50px;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .languages {
+    margin-top: -10px;
+  }
+
+  .action {
+    margin-top: 50px;
+  }
+
+  .card:nth-child(2) {
+    flex-direction: row-reverse;
+  }
+
+  .card:nth-child(4) {
+    flex-direction: row-reverse;
+  }
+
+  .About-my-self {
+    top: 3500px;
+    left: 0;
+    width: 100vw;
+    box-sizing: border-box;
+    border-top-right-radius: 100px;
+    display: block;
+    height: 600px;
+  }
+
+  .top {
+    width: 40%;
+    height: 500px;
+  }
+
+  .About-my-self .About {
+    width: 200px;
+    height: 100px;
+    margin-left: 50px;
+  }
```

Comment:
Kindly make sure your "About myself" is responsive above 768px. You can follow a proportion of 40/60 like I marked below. So they respond to different screen sizes proportionally. 
<img width="660" alt="Screenshot 2023-01-26 at 1 54 32 AM" src="https://user-images.githubusercontent.com/25887253/214712943-6e68382b-1189-4dbd-842e-e7825f401705.png">

<img width="930" alt="Screenshot 2023-01-26 at 2 10 01 AM" src="https://user-images.githubusercontent.com/25887253/214712891-a6ef0d3c-c5c4-4a7b-8893-2f65355a5ec2.png">